modules/ipi-install-configure-network-components-to-run-on-the-control-plane.adoc

```diff
@@ -0,0 +1,126 @@
+// This is included in the following assemblies:
+//
+// ipi-install-configuration-files.adoc
+[id='configure-network-components-to-run-on-the-control-plane_{context}']
+
+= Configure network components to run on the control plane
+
+Configure networking components to run exclusively on the control plane nodes. By default, {product-title} allows any node in the Machine Config Pool to host the `apiVIP` and `ingressVIP` virtual IP addresses. However, many environments deploy worker nodes in separate subnets from the control plane nodes. Consequently, you must place the `apiVIP` and `ingressVIP` virtual IP addresses exclusively with the control plane nodes.
```

Comment:
We recently updated our guidance in how we [use general references to objects](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#general-references), so could you make a small change?

s/Machine Config Pool/machine config pool